src/wiz.c

```diff
@@ -1069,266 +1051,75 @@ do_usage(dbref player)
 #endif /* !NO_USAGE_COMMAND */
 
 /**
- * @TODO: These "topprofs" calls are almost all copy/paste.  I've noted in
- *        other TODO's such as under do_muf_topprofs that there is a lot
- *        of duplicate code ... but these calls are almost 100% duplicated,
- *        so maybe we can refactor them to use a common core function.
- */
-
-/**
- * Implementation of the \@muftops command
- *
- * This shows statistics about programs that have been running.  These
- * statistics are shown for top 'arg1' number of programs.  The default
- * is '10'.  'reset' can also be passed to reset the statistic numbers.
- *
- * This iterates over the entire DB and puts the programs in a
- * 'struct profnode' linked list.  For large numbers of 'arg1', this is
- * really inefficient since its sorting a linked list.
- *
- * This does not do any permission checking.
- *
- * @param player the player doing the call
- * @param arg1 Either a string containing a number, the word "reset", or ""
+ * Adds a 'struct profnode' to a given ordered linked list, if the size
+ * limit has not been reached.
+ *
```

Comment:
I think `@private` has to be added to static functions

Reply:
Thanks! New commit incoming.